vnext/JSI/Shared/NapiJsiV8RuntimeHolder.cpp

```diff
@@ -0,0 +1,93 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#include "NapiJsiV8RuntimeHolder.h"
+
+// V8-JSI
+#include <V8JsiRuntime.h>
+
+using namespace facebook::jsi;
+using namespace facebook::react;
+
+using std::shared_ptr;
+using std::unique_ptr;
+
+namespace Microsoft::JSI {
+
+// See napi_ext_schedule_task_callback definition.
+/*static*/ void NapiJsiV8RuntimeHolder::ScheduleTaskCallback(
+    napi_env env,
+    napi_ext_task_callback taskCb,
+    void *taskData,
+    uint32_t delayMs,
+    napi_finalize finalizeCb,
+    void *finalizeint) {
+
+  NapiJsiV8RuntimeHolder *holder;
+  auto result = napi_get_instance_data(env, (void **)&holder);
+  if (result != napi_status::napi_ok) {
+    //TODO: Signal error
+    return;
+  }
+
+  //TODO: ABI-safe?
```

Comment:
>//TODO: [](http://example.com/codeflow?start=2&length=7)

No, it is not ABI safe.

Reply:
I was trying to add the correct implementation into this PR, but I do not have permissions.
I have added it into my fork of the same branch:
https://github.com/vmoroz/react-native-windows/commit/a91133d7c8aa8883544c8fe1e523af405094d170

Reply:
Integrated.